example/se/PrivateUnsecuredLoanApplicationCreated.json

```diff
@@ -5,17 +5,17 @@
     "source" : "/mycontext",
     "eventID" : "C234-1234-1234",
     "eventTime" : "2018-04-05T17:31:00Z",
-    "extensions" : {
-      "comExampleExtension" : "value"
-    },
+	"extensions":{
```

Comment:
Not part of the spec, it shouldn't be there

Reply:
i'm quite sure its part of cloud event specification https://github.com/cloudevents/spec/blob/v0.1/spec.md#extensions.

The example should showcase how a PrivateUnsecruredLoanApplicationCreated event looks like. Right now all events of that type has an attribute called extensions with another attribute called destination.